src/neo-vm/ExecutionEngine.cs

```diff
@@ -866,6 +866,25 @@ private void ExecuteInstruction()
                         Push(Pop().GetInteger().Sqrt());
                         break;
                     }
+                case OpCode.MODMUL:
+                    {
+                        var modulus = Pop().GetInteger();
+                        var x2 = Pop().GetInteger();
+                        var x1 = Pop().GetInteger();
+                        Push(x1 * x2 % modulus);
```

Comment:
does GetInteger() allow giant numbers greater than 32 bytes? I think it's important to test:
MODMUL 0xff...ff (2^256) x 0xff...ff (2^256) % 0xff...ff (2^256) , to make sure it works...
and also test 2^256+1 scenarios, to make sure it fails...

Reply:
> does GetInteger() allow giant numbers greater than 32 bytes?

No.